examples/nodejs/eventbridge/README.md

```diff
@@ -0,0 +1,54 @@
+# Welcome to Momento <-> Eventbridge Integration Example Project
+
+The project demonstrates a write-through cache pattern for DynamoDB using DynamoDB Streams, AWS EventBridge and Momento.
+
+### **Prerequisites:**
+
+- Momento API Key, can be created using [momento console](https://console.gomomento.com/) if you haven’t already created one
+- [Token Vending Machine](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/nodejs/token-vending-machine) deployed to AWS account
+- AWS Account AccessId and Secret Key
```

Comment:
nit but can you fill me in a bit on how this works? I guess you're building a static app and the browser is talking directly to AWS, so you have to give it AWS creds this way?

Reply:
AWS AccessId and SecretKey are required for creating the dynamodb client so that the browser can directly talk to AWS